README.md

```diff
@@ -1,15 +1,15 @@
 ![](https://img.shields.io/badge/Microverse-blueviolet)
 
 # Project Name
+capstone1-concert-website
 
 > Description the project.
 
 
 ## Built With
 
-- Major languages
-- Frameworks
-- Technologies used
+- HTML CSS and javascript
+- Sass and bootstrap
 
 ## Live Demo (if available)
 
```

Comment:
- [ ] Kindly add the steps it takes for anyone visiting your repo to get the local copy of your project to his/her local machine and also running it 👍🏽

Reply:
- [ ] Kindly make sure you detail each section in your README.
`set up, prerequisite, installation, usage, deployment`
`hint: You can check` [this](https://github.com/christianbiring1/Capstone_project-module1#:~:text=README.md-,Capstone_project%2Dmodule1,-Description%20of%20the) `sample to get the general idea of what you are required to do`